src/gui/controllers/LoginController.java

```diff
@@ -19,7 +19,7 @@ public void initialize(URL arg0, ResourceBundle arg1) {
 	}
 
 	@FXML public void login(ActionEvent event) {
-		Utils.createWindow(null, LoginController.this, "../fxml/Home.fxml", "TasksFX", null, "../css/home.css");
+		Utils.createWindow(null, LoginController.this, "../fxml/Home.fxml", "TasksFX", null, "../css/home.css", "resources.i18n.login");
```

Comment:
This should be `resources.i18n.home` because we are creating the Home Controller here.